adapters/Pangle/PangleAdapter/GADMAdapterPangleUtils.h

```diff
@@ -0,0 +1,13 @@
+//
+//  GADMAdapterPangleUtils.h
+//  PangleAdpter
+//
+//  Created by bytedance on 2021/12/16.
+//
+
+#import "GADMediationAdapterPangle.h"
+
+#define PangleLog(format, args...)    NSLog(@"PangleAdaper | "format,##args)
```

Comment:
Nit: Should we make it a util method like `GADMAdapterPangleErrorWithCodeAndDescription` instead?

Reply:
"GADMAdapterPangleErrorWithCodeAndDescription" uses feedback error, "PangleLog" just some warning and does not terminate the process